Can't resolve 'react-native-web/dist/exports/AppRegistry'

```
Module not found: Can't resolve 'react-native-web/dist/exports/AppRegistry' 
in '/tmp/expo-template-scala/node_modules/expo/build/environment'
```

I get this error, when I try run the app.

` expo start --web`

`yarn add react-native-web react-dom expo`


```
 web  Failed to compile.
/tmp/expo-template-scala/node_modules/expo/build/Notifications/Notifications.js
Module not found: Can't resolve 'react-native-web/dist/exports/AsyncStorage' in '/tmp/expo-template-scala/node_modules/expo/build/Notifications'
```

--- a/App.js
+++ b/App.js
@@ -1,8 +1,9 @@
 import React from 'react';
 import {Text, View} from 'react-native';
 
+let app;
 if (__DEV__) {
-  module.exports = require("./target/scala-2.13/app-fastopt.js").app;
+  app = require("./target/scala-2.13/app-fastopt.js").app;
 } else {
   module.exports = () => {
     return <View style={{flex: 1, justifyContent: "center", alignItems: "center"}}>
@@ -13,5 +14,7 @@ if (__DEV__) {
   }
 
   // uncomment the following line to enable opt building
-  // module.exports = require("./target/scala-2.12/app-opt.js").app;
+  // app = require("./target/scala-2.13/app-opt.js").app;
 }
+
+export default app;

--- a/README.md
+++ b/README.md
@@ -29,7 +29,7 @@ To create a minified app, first build an optimized Scala.js bundle
 $ sbt fullOptJS
 ```
 
-Then, go to `App.js` and uncomment `module.exports = require("./target/scala-2.12/app-opt.js").app;` in order to enable the production bundle. Finally, switch Expo to production mode to load your production app.
+Then, go to `App.js` and uncomment `app = require("./target/scala-2.13/app-opt.js").app;` in order to enable the production bundle. Finally, switch Expo to production mode to load your production app.
 
 ## Ejecting
 If you need to develop your app without depending on Expo, you can eject at any time by running `npm run eject`. This gives you the option to either eject to a "bare" project that uses React Native's build tooling or to an unmanaged ExpoKit project that still uses Expo but lets you add in custom native dependencies.

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
     "react": "16.9.0",
     "react-dom": "16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
+		"react-native-web": "^0.13.14",
     "react-proxy": "1.1.8"
   },
   "devDependencies": {

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "react": "16.9.0",
     "react-dom": "16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
-		"react-native-web": "^0.13.14",
+    "react-native-web": "^0.13.14",
     "react-proxy": "1.1.8"
   },
   "devDependencies": {

